critiquebrainz/frontend/templates/review/browse.html

```diff
@@ -2,32 +2,79 @@
 {% from 'macros.html' import cover_art, review_credit with context %}
 
 {% block title %}{{ _('Reviews') }} - CritiqueBrainz{% endblock %}
-
+{% set type = 'release-group' %}
 {% block content %}
 <h2>{{ _('Reviews') }}</h2>
 <p><em class="text-muted">{{ _('Ordered by creation date') }}</em></p>
 
-<div id="reviews-browse" class="row">
-  {% for review in reviews %}
-    {% set entity = entities[review.entity_id] %}
-    <div class="col-md-4 review">
-      <div class="cover-art-container pull-left">
-        <a href="{{ url_for('review.entity', id=review.id) }}">
-          {{ cover_art(entity.id, review.entity_type, overlay_type=True) }}
-        </a>
+<div id="content">
+  <ul id="tabs" class="nav nav-tabs" data-tabs="tabs">
+    {% if count_release_group > (page-1)*limit %}
```

Comment:
I think it's better if we show all options no matter if there are any reviews there. If there aren't, just tell that explicitly.